package.json

```diff
@@ -31,6 +31,7 @@
     "babel-preset-react": "~6.11.1",
     "babel-preset-stage-0": "~6.5.0",
     "babel-register": "~6.14.0",
+    "css-loader": "^0.25.0",
```

Comment:
As discussed with @fzaninotto, we decided to still use CSS loader, but to embed all styles in `<head>` tag, thanks to the Webpack `style-loader`.